bleak/backends/bluezdbus/utils.py

```diff
@@ -6,8 +6,8 @@
 from bleak.backends.bluezdbus import defs
 from bleak.exc import BleakError
 
-_mac_address_regex = re.compile("^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})$")
-_hci_device_regex = re.compile("^hci(\d+)$")
+_mac_address_regex = re.compile(r'^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})$')
```

Comment:
This row does not need to be changed. The code formatter `black`, which I use for bleak, will change your single quotes to doubles very soon. The string also does not need to be binary. I.e. Remove this change.

Reply:
done